src/descriptors/power_system_structs.json

```diff
@@ -369,43 +442,68 @@
         "data_type": "Bool"
       },
       {
-        "null_value": "Arch(Bus(nothing), Bus(nothing))",
-        "name": "arch",
-        "data_type": "Arch"
+        "name": "activepower_flow",
+        "null_value": "0.0",
+        "data_type": "Float64"
       },
       {
+        "name": "reactivepower_flow",
         "null_value": "0.0",
-        "name": "r",
         "data_type": "Float64"
       },
+      {
+        "null_value": "Arc(Bus(nothing), Bus(nothing))",
+        "name": "arc",
+        "data_type": "Arc"
+      },
+      {
+        "null_value": "0.0",
+        "name": "r",
+        "data_type": "Float64",
+        "comment": "System per-unit value",
+        "valid_range": {"min":0.0, "max":2.0},
+        "validation_action": "error"
+      },
       {
         "null_value": "0.0",
         "name": "x",
-        "data_type": "Float64"
+        "data_type": "Float64",
+        "comment": "System per-unit value",
+        "valid_range": {"min":0.0, "max":2.0},
+        "validation_action": "error"
       },
       {
         "name": "primaryshunt",
         "null_value": "0.0",
-        "data_type": "Float64"
+        "data_type": "Float64",
+        "Comment": "System per-unit value",
+        "valid_range": {"min":0.0, "max":2.0},
+        "validation_action": "error"
       },
       {
         "name": "tap",
         "null_value": "1.0",
-        "data_type": "Float64"
+        "data_type": "Float64",
+        "valid_range": {"min":0.0, "max":2.0},
+        "validation_action": "error"
       },
       {
-        "name": "α",
+        "name": "\u03B1",
```

Comment:
Should we just call this alpha?

Reply:
To be clear this comment is about names exposed in our public API.  Some users might not like being forced to type these names as Greek symbols.